packages/http-server/src/server.ts

```diff
@@ -52,11 +53,12 @@ export const createServer = (operations: IHttpOperation[], opts: IPrismHttpServe
     return done(undefined, payload);
   });
 
-  const mergedConfig = configMergerFactory(
-    { cors: false, mock: { dynamic: false }, validateRequest: true, validateResponse: true },
-    config,
-    getHttpConfigFromRequest,
-  );
+  const mergedConfig = defaults<Partial<IHttpConfig>, IHttpConfig>(config, {
+    cors: true,
```

Comment:
Are we intentionally changing cors from false to true here?

Reply:
Yes! `cors` is enabled by default but the whole merger mechanism was hiding this automatically injecting stuff from 300 different places and functions executed here and there. By removing this and providing an explicit path, these errors popped up quickly and have been solved.

Reply:
Done in c4cfd24fcaac0e58a92bda907d519e5c1bf77064